tower-request-modifier/tests/request_modification.rs

```diff
@@ -3,15 +3,17 @@ extern crate tower_request_modifier;
 extern crate tower_service;
 extern crate tower_test;
 
+use futures_test::task::noop_context;
```

Comment:
we should attempt to use `tokio-test` here, should be possible to get a context, you can peek at the other tower tests for that.